gtsam/hybrid/DCSAM.cpp

```diff
@@ -0,0 +1,167 @@
+/* ----------------------------------------------------------------------------
+
+ * GTSAM Copyright 2010, Georgia Tech Research Corporation,
+ * Atlanta, Georgia 30332-0415
+ * All Rights Reserved
+ * Authors: Frank Dellaert, et al. (see THANKS for the full author list)
+
+ * See LICENSE for the license information
+
+ * -------------------------------------------------------------------------- */
+
+/**
+ * @file    DCSAM.cpp
+ * @brief   Discrete-Continuous Smoothing and Mapping for Factored Models
+ * @author  Kevin Doherty
+ * @author  Varun Agrawal
+ * @date    March 2025
+ */
+
+#include <gtsam/hybrid/DCSAM.h>
+#include <gtsam/hybrid/DiscreteBoundaryFactor.h>
```

Comment:
Again, not sure if you guys use the Google style guide, but if so, consider [including what you use](https://google.github.io/styleguide/cppguide.html#Include_What_You_Use) rather than relying on transitive includes here.